standalone/content/js/standaloneRoomViews.jsx

```diff
@@ -788,6 +802,84 @@ loop.standaloneRoomViews = (function(mozL10n) {
     }
   });
 
+  var Slide = React.createClass({
+    propTypes: {
+      imageClass: React.PropTypes.string.isRequired,
+      indexClass: React.PropTypes.string.isRequired,
+      joinRoom: React.PropTypes.func.isRequired,
+      text: React.PropTypes.string.isRequired,
+      title: React.PropTypes.string.isRequired
+    },
+
+    handleGotItClick: function(event) {
+      event.stopPropagation();
+      event.preventDefault();
+      var overlayElem = document.getElementsByClassName("intro-overlay")[0];
+      overlayElem.classList.add("hide");
+      localStorage.setItem("introSeen", true);
+      this.props.joinRoom();
+    },
+
+    render: function() {
+      return (
+        <div className="slide">
+          <div className={this.props.indexClass}>
+            <div className="slide-layout">
+              <div className={this.props.imageClass} />
+              <h2>{this.props.title}</h2>
+              <div className="slide-text">{this.props.text}</div>
+              <button className="button-got-it" onClick={this.handleGotItClick}>
+                {mozL10n.get("intro_slide_button")}
+              </button>
+            </div>
+          </div>
+        </div>
+      );
+    }
+  });
+
+  var IntroOverlayView = React.createClass({
+    propTypes: {
+      joinRoom: React.PropTypes.func.isRequired
+    },
+
+
+    handleCloseClick: function(event) {
+      event.stopPropagation();
+      event.preventDefault();
+      // hide or remove the
+      var overlayElem = document.getElementsByClassName("intro-overlay")[0];
+      overlayElem.classList.add("hide");
+      this.props.joinRoom();
+    },
+
+    render: function() {
+      var slideNode = {
+        id: "slide1",
+        imageClass: "slide1-image",
+        title: mozL10n.get("intro_slide_title"),
+        text: mozL10n.get("intro_slide_copy", {
+          clientSuperShortname: mozL10n.get("clientSuperShortname")
```

Comment:
Excitingly, clientSuperShortname is currently only defined in add-on.properties for the browser add-on, rather than in shared.properties for everyone.  So we technically _have_ the translated string we need, but it's not in the right places... Hmmm.  One thing that comes to mind is something like a sed script, assuming all the translations are already done (are they?).  We could also choose to package up add-on.properties with the standalone temporarily.  @Standard8: thoughts?